cfpb/capital-framework
# Removed all the duplicate icon rules

The latest changes to cf-icons made the duplicate icon rules for links and buttons unnecessary.

## Removals

- Duplicate icon code from cf-buttons
- Duplicate icon code from cf-typography

## Testing

- checkout this branch, be sure you've npm linked these modules by running `npm run cf-link`
- checkout the `example/remove-icons` branch within the sandbox
- in that repo run `npm run cf-link && npm run build && npm start`
- navigate to `http://localhost:3000/components/cf-buttons/` and check the __Icon Buttons__ section
- navigate to `http://localhost:3000/components/cf-typography/` and check out the __Links__ section

## Review

- @Scotchester 

## Screenshots

Everything should look identical to before.

## Checklist
